### PR TITLE
possible fix for null stringValue

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
I couldn't repro this issue but this might be some timing related problem

### CODE REVIEW

File: shared/login/signup/success/index.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ import {connect} from 'react-redux'
...
 
-// $FlowIssue
+import type {TypedState} from '../../../constants/reducer'
+import type {TypedDispatch} from '../../../constants/types/flux'
+
 export default connect(
-  state => ({
-    paperkey: state.signup.paperkey,
+  (state: TypedState) => ({
+    paperkey: state.signup.paperkey && state.signup.paperkey.stringValue('') || '',
```
Comment (MarcoPolo) [inline shared/login/signup/success/index.js]: this doesn't take an argument

**Comment by cjb** [comment]
(Please hold off on this one -- I have a repro and this won't fix it.)

File: shared/login/signup/success/index.js (hunk 1 above)
Comment (chrisnojima) [inline shared/login/signup/success/index.js]: yeah that was me fixing a flow thing in theh wrong place

**Comment by chrisnojima** [comment]
killing this

**Pull request closed without merging**